extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.js

```diff
@@ -191,4 +188,27 @@ export class AmpStoryDevToolsTabLogs extends AMP.BaseElement {
     title.appendChild(statusText);
     return title;
   }
+
+  /**
+   * Updates the icon (passed / failed) next to the logs tab selector
+   * @param {!Array} errorList
+   */
+  updateLogsTabIcon(errorList) {
+    const logsTabSelector = this.win.document.querySelector(
+      '[data-tab="Logs"]'
+    );
+    if (logsTabSelector) {
+      this.mutateElement(() => {
+        if (errorList.length) {
+          const errorNumberEl = htmlFor(
+            this.element
+          )`<span class="i-amphtml-story-dev-tools-log-status-number-failed">0</span>`;
+          errorNumberEl.textContent = errorList.length;
+          logsTabSelector.appendChild(errorNumberEl);
+        } else {
```

Comment:
That's 3 levels of indentation, plz slightly rewrite this part to keep only 1, eg:

```
if (!logsTabSelector) {
  return;
}

if () {
   ... // generates HTML
   this.mutateElement(() => append);
}
...
```